tests/test_zip.py

```diff
@@ -0,0 +1,34 @@
+import glob
+import os.path
+import subprocess
+import tempfile
+import unittest
+
+
+_TEST_SCRIPT='; '.join('''import certifi
+import os.path
+assert os.path.exists(certifi.where()), "certs do not exist"
+print "PASS"'''.split('\n'))
```

Comment:
The `python` binary could just as easily be linked to `python3` in a virtualenv. Please use `print("PASS")`